src/helper_addressGenerator.py

```diff
@@ -0,0 +1,52 @@
+"""
+Create random address
+"""
+
+import time
+import defaults
+import queues
+import state
+
+from bmconfigparser import BMConfigParser
+
+
+def getSortedAccounts():
+    """Get a sorted list of configSections"""
+
+    configSections = BMConfigParser().addresses()
+    configSections.sort(
+        cmp=lambda x, y: cmp(
+            unicode(
+                BMConfigParser().get(
+                    x,
+                    'label'),
+                'utf-8').lower(),
+            unicode(
+                BMConfigParser().get(
+                    y,
+                    'label'),
+                'utf-8').lower()))
+    return configSections
+
+
+def checkHasNormalAddress():
+    """method for checking address"""
+    for address in getSortedAccounts():
```

Comment:
no need for sorting, just `BMConfigParser().addresses`.

Reply:
ok